pkg/cloud/instance.go

```diff
@@ -139,6 +138,37 @@ func (c *client) ResolveTemplate(
 	return templateID, nil
 }
 
+func (c *client) ResolveDiskOffering(csMachine *infrav1.CloudStackMachine) (diskOfferingID string, retErr error) {
```

Comment:
Please add a comment above all functions/methods that are exported.

``` Go
\\ ResolveDiskOffering resolves to make disk offerrings the best thing ever!
func (c *client) ResolveDiskOffering(csMachine *infrav1.CloudStackMachine) (diskOfferingID string, retErr error) {
...
```
There's a linter for this somewhere, but I guess we've not got it turned  on as we still have quite a few without comments.

Personally, I put them there whether they're exported or not. : )